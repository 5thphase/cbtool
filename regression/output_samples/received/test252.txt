 status: Sending a termination request for vm_17 (cloud-assigned uuid 746217EE-05D1-5E61-821E-8AD755044DB3)....
 status: Sending a termination request for vm_14 (cloud-assigned uuid 11111111-1111-1111-1111-111111111111)....
 status: Sending a termination request for vm_21 (cloud-assigned uuid 873D9A2F-927E-55EF-8365-710CA41CA756)....
 status: Sending a termination request for vm_6 (cloud-assigned uuid 8BB22938-11DA-5E10-8A01-7729DF03BF44)....
 status: Sending a termination request for vm_11 (cloud-assigned uuid 841725C0-6AA7-5C63-8FCD-18F4DC730D38)....
 status: Sending a termination request for vm_20 (cloud-assigned uuid DA6DBF12-5DCE-5194-895F-16DD417B75A1)....
 status: Sending a termination request for vm_9 (cloud-assigned uuid CE20EE9C-D56A-5C43-B037-E428C80E0D61)....
 status: Sending a termination request for vm_15 (cloud-assigned uuid 6840F186-48BA-5199-8DDF-6E656DCD9B71)....
 status: Sending a termination request for vm_13 (cloud-assigned uuid 9CF8FFBB-4E79-5741-A9D8-57089C722CB4)....
 status: Sending a termination request for vm_8 (cloud-assigned uuid 9B675A7D-3509-55A3-938E-9C43D893B029)....
 status: Sending a termination request for vm_10 (cloud-assigned uuid AB945E48-D052-5D2F-9172-D8C1FDE9DF26)....
 status: Sending a termination request for vm_19 (cloud-assigned uuid A10C614A-70CD-59F2-AF8B-A20B024A7FD2)....
 status: Sending a termination request for vm_18 (cloud-assigned uuid 114CDAE1-A7FD-5EDD-B40A-A69D9F783ACD)....
 status: Sending a termination request for vm_12 (cloud-assigned uuid 6B78A8CA-FB7C-5C39-97E7-8B97802623D9)....
 status: Sending a termination request for vm_3 (cloud-assigned uuid 8EFD92C6-7E2E-5099-B59D-6D675BDB0DED)....
 status: Sending a termination request for vm_16 (cloud-assigned uuid 096A1DD0-44C0-56C3-B5FE-908CEDE127E4)....
VM object undefined (named "all") was sucessfully detached from this experiment.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
No objects available.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |2.07M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |1881542                       
|Total Connections Received                           |6067                          
|Total Commands Processed                             |1662473                       
|Number of Keys                                       |324                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.6                         
|Storage Size                                         |823296                        
|Data Size                                            |150672                        
|Index Size                                           |294336                        
|Average Object Size                                  |389.333333333                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |22                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |132                           

 ---------------- VMC-WIDE COUNTERS ----------------
|B25B0238-8257-5847-B4B0-AE0622BDBB60 (simzone_d) VM RESERVATIONS|0                             
|A7ADD8B8-3B77-5C04-A200-4E10FD89DD12 (simzone_b) VM RESERVATIONS|0                             
|21F066AD-6280-52AA-956C-847B043FD63E (simzone_c) VM RESERVATIONS|0                             
|2B8B3052-BD8E-5E84-8885-8B75FD72F9AB (simzone_a) VM RESERVATIONS|0                             

