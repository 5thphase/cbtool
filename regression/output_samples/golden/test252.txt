 status: Sending a termination request for vm_3 (cloud-assigned uuid 68F9CCCC-8F12-556D-B8FE-33B12798A737)....
 status: Sending a termination request for vm_9 (cloud-assigned uuid A7DA1ADE-A671-5BB4-8F83-75AC7177D26C)....
 status: Sending a termination request for vm_20 (cloud-assigned uuid 1443F7DD-3509-562E-A2B1-84C12797BF25)....
 status: Sending a termination request for vm_8 (cloud-assigned uuid F6FF7D5E-FD5A-5157-8E4B-740759F2F596)....
 status: Sending a termination request for vm_6 (cloud-assigned uuid 2EF34503-8ED7-5E34-A8D6-4A02070B7976)....
 status: Sending a termination request for vm_7 (cloud-assigned uuid EF8D9B2F-99D0-5C9D-9207-E33B1C27A13E)....
 status: Sending a termination request for vm_13 (cloud-assigned uuid 654BAD35-65E1-5A9E-87D8-7F84DCC1F47B)....
 status: Sending a termination request for vm_14 (cloud-assigned uuid 11111111-1111-1111-1111-111111111111)....
 status: Sending a termination request for vm_18 (cloud-assigned uuid A7640197-72D4-54B8-A063-8CFE0BDB9421)....
 status: Sending a termination request for vm_17 (cloud-assigned uuid DF47EA94-677F-5178-9E35-531A1EF41B8C)....
 status: Sending a termination request for vm_21 (cloud-assigned uuid 43CE0E93-F8B2-543B-9866-26D928430322)....
 status: Sending a termination request for vm_11 (cloud-assigned uuid D06136C0-FC4A-5C33-A387-318E3FA5B525)....
 status: Sending a termination request for vm_15 (cloud-assigned uuid FD16BEB3-7F3B-50F0-8BA1-3C7962E51F06)....
 status: Sending a termination request for vm_10 (cloud-assigned uuid 4AAF73D7-B95D-5A25-BF15-B0EC538B2655)....
 status: Sending a termination request for vm_16 (cloud-assigned uuid 4F88EF1E-8B53-561D-A634-05A072ECB438)....
 status: Sending a termination request for vm_19 (cloud-assigned uuid 554CF820-6D4D-5353-9E42-81B2D1654B75)....
VM object undefined (named "all") was sucessfully detached from this experiment.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
No objects available.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |2.73M                         
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |2416636                       
|Total Connections Received                           |4455                          
|Total Commands Processed                             |3665075                       
|Number of Keys                                       |324                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |5447958528.0                  
|Data Size                                            |4960722792.0                  
|Index Size                                           |133620368                     
|Average Object Size                                  |8110.48731368                 
|Collections                                          |40                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |22                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |132                           

 ---------------- VMC-WIDE COUNTERS ----------------
|2A3C7A6F-F507-5C26-894B-CA777DA4A067 (simzone_a) VM RESERVATIONS|0                             
|841649F1-B9B9-5F66-BF08-8188BB2F57BF (simzone_c) VM RESERVATIONS|0                             
|290AA171-A379-5C01-9211-06AFF9C329FA (simzone_d) VM RESERVATIONS|0                             
|DDE459C8-87C4-5C23-B346-48A22B829E97 (simzone_b) VM RESERVATIONS|0                             

